CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 The example configuration file contained an invalid section `vast`.
+   This has been changed to the correct name `system`.
```

Comment:
Please add ` [#706](https://github.com/tenzir/vast/pull/706)` to both entries.

Reply:
I opted to to link to #705 for the config file fix, since that PR contains the actual change.